fluent-react/src/localized.js

```diff
@@ -82,7 +82,8 @@ export default class Localized extends Component {
   render() {
     const { l10n } = this.context;
     const { id, attrs, children } = this.props;
-    const elem = Children.only(children);
+    const elem = typeof(children) !== "undefined" ?
+      Children.only(children) : null;
```

Comment:
We should still allow fallback strings in the JSX:

```jsx
<Localized id="hello">
    Hello, world!
</Localized>
```

In such cases, `Localized` still receives a single child.